pages/common/asciinema.md

```diff
@@ -0,0 +1,35 @@
+# asciinema
+
+> Record and replay terminal sessions, and optionally share them on asciinema.org.
+
+- Associate the local install of `asciinema` with an asciinema.org account:
+
+`asciinema auth`
+
+- Make a new recording (to be uploaded or saved locally):
+
+`asciinema rec`
+
+- Make a new recording and save it to a local file:
```

Comment:
So where does it save when no file params are passed ?

Reply:
It asks when the recording is ended. Instructions are provided interactively to start, stop and save/upload the cast in the relevant moments of the capture process.

Do you think there's a way to make this clearer (without making the line too long, that is)?

Reply:
I gave it a try, check it out :)